.github/workflows/firebase-distribution.yml

```
@@ -0,0 +1,13 @@
+on: push
+jobs:
+  distribute:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/firebase-distribution
+        with:
+          x: 13
+          y: 3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)